### ISSUE TITLE
Explain how to use it on the website

### ISSUE THREAD

**Issue by gbinal** [project-open-data/datajson-builder#1]
When on https://project-open-data.cio.gov/datajson-builder/, I don't readily see how to actually use the tool.

**Comment by JJediny** [comment]
Assigning myself to build out a blog like listview & add documentation on emulating how to create a _dataset directory and data.json for use on any Jekyll template...

**Comment by JJediny** [comment]
To be clear of intention... The concept here is that any agency could quickly standup a light-weight, no-security headache, website immediately where each "blog/page" === a dataset -> all posts/pages collectively then automatically compile together in single data.json file that can be harvested by data.gov... Where the github pages 'front matter' (ie YAML attributes) can concurrently feature/filter content for the agencies specfic need and as a indirect/secondary effect compile valid data.json v1.1

**Comment by pjdufour** [comment]
Yeah, we'll want to add dataset "detail pages".  The MVP to start the conversation was just the data.json template.
